src/utils/http/basicHttpClient.ts

```diff
@@ -230,7 +230,7 @@ export enum HttpMethod {
   Post = 'post',
   Put = 'put',
   Delete = 'delete',
-  Patch = 'patch',
+  Patch = 'PATCH',
```

Comment:
We should capitalize on all verbs here. I see in `5.1.1` that it is indeed case-sensitive and should be capitalized: https://www.w3.org/Protocols/rfc2616/rfc2616-sec5.html#sec5.1.1